conf/report/filter_ui_test.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+import sys
+
+def setDriver(driver=None):
+    if driver == None:
+        return webdriver.Chrome()
+    else:
+        return driver
+
+def openPage(URL):
+    driver.get(URL)
+    driver.implicitly_wait(10)
+    driver.find_element_by_tag_name('button').click()
+
+def addEntries(iter):
+    for _ in range(iter):
+        driver.find_element_by_class_name('filter-add').click()
+
+def fillEntryType(num, type):
+    entry_type = "//*[@class='filter-entry-type']"
+    select = Select(driver.find_elements_by_xpath(entry_type)[num])
+    select.select_by_value(type)
+
+def fillSpan(num, operator, value, relation=None):
```

Comment:
If you change it to `fillSpan(entry_id, *args)`, loop over `args` and assign each arg's value to a Select queried with an XPath like `f"(//*[@id='filter']/ul/li[{ entry_id + 1 }]/span/select)[{ loop_iteration + 1 }]"`, it will be possible to use it this way: `fillSpan(0, ">", "50", "or", "<", "80")`.